eng/targets/Helix.Common.props

```diff
@@ -27,7 +27,8 @@
 
   <!-- daily scheduled only queues -->
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true'">
-    <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
+    <!-- Temporarily ignoring win7 builds as they are failing on random tests -->
+    <!--HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" /-->
```

Comment:
Maybe just delete the line and file an issue?

Reply:
I can also file an issue maybe, even better for visibility, unless you tell me that triage is super efficient and you won't miss it, then I'll trust you.

Reply:
Well we didn't comment out win8 and there's no issue for that one, so maybe add back win8 then?  just trying to be consistent :)

Reply:
In a commented form I mean for win8

Reply:
/fyi #21421 covers restoring the win8 queue. I plan to address this today or tomorrow

Reply:
Because we haven't tested on win8 w/ the latest SDK, don't restore it unless it gets run in PR validation builds or you hack the PR to include it in your builds.

Reply:
But, please, please, please avoid commented-out code. Just delete.

Reply:
Created the issue. Might require some better labels and other metadata